rest/replicator_test.go

```diff
@@ -4067,6 +4066,7 @@ func TestSGR2TombstoneConflictHandling(t *testing.T) {
 				// Delete doc on remote
 				resp = remotePassiveRT.SendAdminRequest("PUT", "/db/docid2?rev=3-abc", `{"_deleted": true}`)
 				assertStatus(t, resp, http.StatusCreated)
+				requireTombstone(t, tb2, "docid2")
```

Comment:
As discussed, I'm not sure this is adding any blocking - it's just doing an immediate assertion which will either pass or fail.